Document/PostManager.php

```diff
@@ -25,6 +27,8 @@ class PostManager extends BaseDocumentManager implements PostManagerInterface
      * @param $slug
      *
      * @return mixed
+     *
+     * @deprecated since version 3.x, to be removed in 4.0. Please use PostManager::findOneByPermalink instead
```

Comment:
"Use PostManager::findOneByPermalink instead."

Remove the "Please" word. Make it short to be efficient. :+1: